src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -1997,6 +1997,30 @@ TEST_F(EnsembleConfigurationValidationWithCustomNode, NotAllCustomNodeInputsAreC
     ASSERT_EQ(pipelineDefinition->validate(manager), StatusCode::PIPELINE_NOT_ALL_INPUTS_CONNECTED);
 }
 
+TEST_F(EnsembleConfigurationValidationWithCustomNode, CustomNodeMissingOutput) {
+    std::vector<NodeInfo> info{
+        {NodeKind::ENTRY, ENTRY_NODE_NAME, "", std::nullopt, {{pipelineInputName, pipelineInputName}}},
+        {NodeKind::CUSTOM, "custom_node_1", "", std::nullopt, {{"out", "not_existing_output"}}, std::nullopt, {}, mockedLibrary,
+            parameters_t{
+                {"in_InputNumbers", "1,30,7;I32"},
+                {"out_OutputNumbers", "1,2000;FP32"},
+                {"not_existing_output", "1,2000;FP32"}}},
```

Comment:
Why is this line  needed? As far I remember mockedLibrary accepts only parameters with `in_` and `out_` prefix.